src/main/resources/view/UserProfileWindow.fxml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.net.URL?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.Scene?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.stage.Stage?>
+
+<fx:root resizable="false" title="User Profile" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/19" xmlns:fx="http://javafx.com/fxml/1">
+    <icons>
+        <Image url="@/images/user_icon.png" />
```

Comment:
I don't know how I feel about putting a User Icon. The only realistic way users can change the icon would be to manually put it in the folder themselves. There's no way we want to implement an "upload photo" feature.

As a placeholder it's fine and LGTM. Maybe put the NUS logo there =)

Reply:
This isn't for the user to change, its just a icon on the top of the window beside the window name :)